scripts/build/lib.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+set -e
+
+# See https://docs.github.com/en/actions/use-cases-and-examples/deploying/installing-an-apple-certificate-on-macos-runners-for-xcode-development
+function setup_runner() {
+    local app_profile="$1"
+    local app_profile_path="$2"
+    local ne_profile="$3"
+    local ne_profile_path="$4"
+    profiles_path="$HOME/Library/MobileDevice/Provisioning Profiles"
+    keychain_pass=$(openssl rand -base64 32)
+    keychain_path="$(mktemp -d)/app-signing.keychain-db"
+
+    # Select Xcode specified by the workflow
+    sudo xcode-select -s "/Applications/Xcode_$XCODE_VERSION.app"
+
+    # Install provisioning profiles
+    mkdir -p "$profiles_path"
+    install_provisioning_profile \
+        "$profiles_path" \
+        "$app_profile" \
+        "$app_profile_path"
+    install_provisioning_profile \
+        "$profiles_path" \
+        "$ne_profile" \
+        "$ne_profile_path"
+
+    # Create a keychain to use for signing
+    security create-keychain -p "$keychain_pass" "$keychain_path"
+    security default-keychain -s "$keychain_path"
```

Comment:
This is new; needed in newer versions of Xcode for it to find our codesigning certs.